docs/blog/newsletter-october-2023.rst

```diff
@@ -0,0 +1,130 @@
+
+The newsletter this month covers how to include links within your docs, whether you can show some of your personality in your docs, how to handle docs in the same repository as product source code, and how much (or little) to capitalize feature names. Hope you enjoy!
+
+--------------
+Hyperlink Text
+--------------
+
+How to deal with links is a recurring question for documentarians. To answer it, some people suggested guidelines from the `W3C <https://www.w3.org/WAI/tips/writing/#make-link-text-meaningful>`__, `Microsoft <https://learn.microsoft.com/en-us/style-guide/urls-web-addresses>`__, and `Google <https://developers.google.com/style/link-text>`__.
+
+The general consensus was to include text that's relevant — not just ‘click here’. Suggestions included using the linked page’s title, turning the title into relevant text, and creating descriptive text based on the context. Non-writers (such as developers) may prefer using the title for simplicity. More experienced documentarians may want to craft the text.
+
+There are several issues involved:
+
+- If there are inadequate guidelines for cross-referencing content, new or updated guidelines may be needed.
+- The page title may be too long or unhelpful. If so, either edit the title or summarize the content in a few words.
+- Some documentation systems focus on short, succinct topics. This may require many links to cross-reference content adequately.
+
+A related issue concerns *where* to put the links: throughout the content or at the end of the content in a separate list? The main arguments against putting links at the end: the link may have lost its context or the reader may never scroll that far. In either case, it’s important to use relevant text. Having the link within the content may disrupt the reader by triggering cognitive overload and making them wonder: "Should I click or not? Should I click as I come to the link or wait until I finish reading?"
+
+If you put links within the content, `one reference suggested <https://readabilityguidelines.co.uk/content-design/links/#2-avoid-mid-sentence-links>`__ always putting the link at the end of the sentence to minimize distraction. Accessibility was an additional concern, but exact solutions were unclear because interpretation depends upon the screen reader. 
+
+--------------------------------
+Personality in Technical Writing
+--------------------------------
+
+There are still different views among documentarians on the issue of the importance of personality, customization, and tone in technical writing.
+
+Some people said that strict accuracy was the most important in technical writing, while others said that there was room for creativity and humor as long as precision wasn’t compromised. In particular, they said that adding a bit of personality can make such writing more interesting, especially if the readers can handle it.
```

Comment:
"the most important in technical writing" seems like it needs a noun, like "the most important consideration in technical writing"

Reply:
@hillaryfraley  Good call! Alternatively "accuracy was most important in ..."

Reply:
I remember changing it to that and thinking, "Remember to come back and add a noun." 🤦 Thanks!

Reply:
```suggestion
Some people said that strict accuracy was the most important consideration in technical writing, while others said that there was room for creativity and humor as long as precision wasn’t compromised. In particular, they said that adding a bit of personality can make such writing more interesting, especially if the readers can handle it.
```